ebpfapi/Source.def

```diff
@@ -91,6 +91,7 @@ EXPORTS
     ebpf_get_program_type_name
     ebpf_link_close
     ebpf_map_pin
+    ebpf_map_wait_for_change
```

Comment:
Strongly considering removing this API as we can only signal on bpf_map_update_elem/bpf_map_delete_elem, but not on bpf_map_lookup_elem + modify of the value returned.

Reply:
that is a good point, at minimum rename to ebpf_map_wait_for_update which is slightly less bad in that respect